src/core.c/Main.pm6

```diff
@@ -276,7 +276,32 @@ my sub RUN-MAIN(&main, $mainline, :$in-as-argsfiles) {
                     }
                     @positional.push($argument);
                 }
-                @arg-help.push($argument => $param.WHY.contents) if $param.WHY and (@arg-help.grep:{ .key eq $argument}) == Empty;  # Use first defined
+            		#@arg-help.push($argument => $param.WHY.contents) if $param.WHY and (@arg-help.grep:{ .key eq $argument}) == Empty;  # Use first defined
```

Comment:
This line isn't needed

Reply:
The comment was there to serve as a location mark. I will remove it.